app/controllers/branches_controller.rb

```diff
@@ -35,6 +35,7 @@ def show
       # due to the very permissive branch id constraint. Instead users will
       # have to specify a query param of format=rss to receive the RSS feed.
       format.rss { @builds = @builds.reverse } # most recent first
+      format.json { render :json => @builds }
```

Comment:
for the rss feed it kind of made sense to not include any information about the branch. That is not true for the json format. However you can't use `format.json { render json: @branch, include: :builds` because that will give you all the builds on the branch instead of the 12 or so most recent that have been loaded by this method. In order to get the behavior you what you want you'll need to create a `app/views/branches/show.json.erb`.